system/Language/en/View.php

```diff
@@ -17,4 +17,5 @@
     'noCellClass'           => 'No view cell class provided.',
     'invalidCellClass'      => 'Unable to locate view cell class: {0}.',
     'tagSyntaxError'        => 'You have a syntax error in your Parser tags: {0}',
+    'invalidDecoratorClass' => '{0} is not a valid ViewDecorator.',
```

Comment:
If we go with the rename I think this would read better without the `Interface` suffix so I suggest making it natural:
```suggestion
    'invalidDecoratorClass' => '{0} is not a valid View Decorator.',
```